bin/main.rb

```diff
@@ -1,46 +1,105 @@
 #!/usr/bin/env ruby
-def players
-  puts "Hello! who's the player one?, you will play with X."
-  p_one = gets
-  puts "Cool, and who's going to play with O?"
-  p_two = gets
-  puts "Nice to meet you #{p_one.chomp} and #{p_two.chomp}. Lets begin!"
-  [p_one, p_two]
+class Player
+  attr_accessor :name, :char
+  def initialize(char)
+    @char = char
+  end
+
+  def start
+    puts "Hello, what's your player name?"
+    @name = gets
+    puts "#{@name.chomp}, you will play as #{@char} \n\n"
+    puts
+  end
+end
+
+first = Player.new('X')
+first.start
+second = Player.new('O')
+second.start
+
+class GameDisplay
+  attr_accessor :position, :pos, :repeated
```

Comment:
[Please fix] I don't find the `:repeated` accessor used anywhere, please remove it if you don't use it.